src/components/Book.js

```diff
@@ -1,3 +1,21 @@
-function Book() {}
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-one-expression-per-line */
```

Comment:
[OPTIONAL] Rather than disable the linter, strive to fix them as this makes you more accustomed to the style guide of the tech.

Reply:
Well noted. I will work on it. 
Thanks :smile: